config/webpack.config.js

```diff
@@ -125,6 +130,11 @@ webpackConfig.module.loaders = [{
 
 // ------------------------------------
 // Style Loaders
+//
+// style    - This loader injects <style> tags.
+// css      - Resolves imports and returns css.
+// postcss  - Applies PostCSS processing.
+// sass     - Loads and compiles sass files.
```

Comment:
:+1: It's probably worth noting too (and this is a bit of an assumption, but I think it's correct) that loaders are executed in reverse order when declared in an array like this. So sass -> postcss -> css -> style.

I actually didn't realize that before.